### PR TITLE
November news

### PR DESCRIPTION
**Pull request by maelle**
TODOS

- [x] check video link
- [x] check no use case on the forum, delete section

### CODE REVIEW

File: content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+<!-- to be curated manually -->
+
```
Comment (yabellini) [inline content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd:45]: ## Multilingual Publishing 

Open Source and Open Science are global movements, but most of their material and resources are published in English. Non-English speakers face a significant barrier to being part of these movements.  Publishing multilingual resources can lower the barrier to access to knowledge, help democratize access to quality resources and increase the possibilities of contributing to software and open science projects.

With the the support of the [Chan-Zuckerberg Initiative](https://ropensci.org/blog/2021/12/20/inclusive-leadership-program/) and [NumFOCUS](https://numfocus.org/) we start to translate rOpenSci’s material on best practices for software development, code review, and contribution to open source projects beyond the English language. As part of this effort we also develop guidelines for translating open source resources to a wider audience.

Learn more about this project and how you can contribute on this blog post.

**Review by yabellini — COMMENTED** [review]
Adding text and placeholders for Community call and multilingual publishing project

File: content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,617 @@
...
+Open Source and Open Science are global movements, but most of their material and resources are published in English. 
+Non-English speakers face a significant barrier to being part of these movements.  
+Publishing multilingual resources can lower the barrier to access to knowledge, help democratize access to quality resources and increase the possibilities of contributing to software and open science projects.
+
+With the the support of the [Chan-Zuckerberg Initiative](/blog/2021/12/20/inclusive-leadership-program/) and [NumFOCUS](https://numfocus.org/) we start to translate rOpenSci’s material on best practices for software development, code review, and contribution to open source projects beyond the English language. 
+As part of this effort we are also developping guidelines for translating open source resources to a wider audience.
+
+Learn more about this project and how you can contribute on this blog post.
+
```
Comment (yabellini) [inline content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd:54]:
```suggestion
## Champions Program

The application period for our champions program is now closed.

We are very excited about the response from the community: we received 74 applications for champions and 28 for mentors from 31 countries.

We are very grateful to those who took the time to apply and all those who helped us spread the word about these calls.

Over the next few weeks, we will review the proposals and select the nominees.  All applicants will receive feedback on their applications.

You can learn more about it on the [program web page](/champions/).

```

Comment (yabellini) [reply content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd:54]: How I add the picture of a map?

**Review by yabellini — COMMENTED** [review]
Maëlle I add some update info about the application on the champions program.  I don't know how I should add a picture.  I would like to share the map with the application countries.

File: content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd (hunk 2 above)
Comment (yabellini) [reply content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd:54]: countriesApplications
Here is the map.  
Alt-text: World Maps with the applicant's countries colored.  The map shows America almost all painted and then some few countries from Asia, Africa, Europe and Oceania.

File: content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,617 @@
...
+## rOpenSci HQ
+
+### Multilingual Publishing 
+
+Open Source and Open Science are global movements, but most of their material and resources are published in English. 
+Non-English speakers face a significant barrier to being part of these movements.  
+Publishing multilingual resources can lower the barrier to access to knowledge, help democratize access to quality resources and increase the possibilities of contributing to software and open science projects.
+
+With the the support of the [Chan-Zuckerberg Initiative](/blog/2021/12/20/inclusive-leadership-program/) and [NumFOCUS](https://numfocus.org/) we start to translate rOpenSci’s material on best practices for software development, code review, and contribution to open source projects beyond the English language.
```
Comment (yabellini) [inline content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd:50]:
```suggestion
With the the support of the [Chan-Zuckerberg Initiative](/blog/2021/12/20/inclusive-leadership-program/),  [NumFOCUS](https://numfocus.org/) and the [RConsortium](https://www.r-consortium.org/), we start to translate rOpenSci’s material on best practices for software development, code review, and contribution to open source projects beyond the English language. 
```

**Review by yabellini — COMMENTED** [review]
Add Rconsortium to the supporters

**Comment by maelle** [comment]
@yabellini could you please give me the image file and its caption + alternative text? This way I can add it and you can see how it works for next time.

**Comment by maelle** [comment]
@yabellini I will send this newsletter on Friday, not today, today was the day I wanted to have a nearly ready draft for @steffilazerte :smile_cat:

File: content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,633 @@
...
+
+* [Tuesday, December 6th, 9 AM Australian Western / 1:00 UTC](/events/coworking-2022-12/) "Getting started with targets!" *Hosted by community host [Nick Tierney](/author/nicholas-tierney/) and [Steffi LaZerte](/author/steffi-lazerte/)*
+    - Dive into the world of targets! Do some studying; Start setting it up for some of your projects;
+    - Ask [Nick](/author/nicholas-tierney/) for suggestions how how to get started or tips and tricks;
+    - Cowork independently on work related to R. Work on packages that tend to be neglected; What ever you need to get done!
+    
+
+## Software :package:
+
+### New packages
```
Comment (steffilazerte) [inline content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd:96]: Missing reviewers for [daiquiri](https://docs.ropensci.org/daiquiri), they were Brad Cannell and Mauro Lepore (I read the review issue for my tweet last week so know the reviewers changed from what is in the main comment).

File: content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,633 @@
...
+Say you have some slow and fragile [tests querying an API](https://books.ropensci.org/http-testing/).
+If you want to run them on continuous integration only,
+refer to [Bryce Mecum's blog post](https://brycemecum.com/2020/08/05/testing-r-api-packages/).
+
+### How to use additional packages for the pkgdown website only
+
+Say a package is needed for a pkgdown article of your package (but not a vignette), or for nicer autolinking of a reference to a function (for instance if you recommend `usethis::create_package()`). 
+Assuming you're building your website with GitHub Actions from [r-lib/actions](https://github.com/r-lib/actions/tree/v2-branch/examples) (that you might have gotten via [usethis](https://usethis.r-lib.org/reference/github_actions.html)).
+
+If so, you can use the `Config/Needs/website` field in `DESCRIPTION`.
```
Comment (steffilazerte) [inline content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd:618]: As always, I learn something new! I always added these packages to the GitHub Actions .yml, I didn't know you could add it to the DESCRIPTION file!

**Review by steffilazerte — COMMENTED** [review]
Looks good! Some wording suggestions for the HQ sections, I added the coworking details and a couple of random comments elsewhere.
And as always, I learned something new, thank you!

File: content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd (hunk 4 above)
Comment (maelle) [reply content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd:103]: thank you!!

File: content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd [hunk 6]
```diff
@@ -0,0 +1,643 @@
...
+* [Tuesday, Jan 10th, 14:00 European Central / 13:00 UTC](/events/coworking-2023-01/) "Working with new R users" *Hosted by community host [Alex Koiter](https://alexkoiter.ca) and [Steffi LaZerte](/author/steffi-lazerte/)*
+    - Brainstorm ideas for supporting and encouraging new R users; Annotate a script for a friend or colleague to help them learn;
+    - Talk to [Alex](https://alexkoiter.ca) and discuss how to share the love of R with new R users.
+
+* [Tuesday, Feb 7th, 9:00 Americas Pacific / 17:00 UTC](/events/coworking-2023-02/) "Setting up Continuous Integration" *Hosted by community host [Hugo Gruson](/author/hugo-gruson/) and [Steffi LaZerte](/author/steffi-lazerte/)*
+    - Do some reading to learn about Continuous Integration; Setup Continuous Integration on one (or more) of your projects;
+    - Talk to [Hugo](/author/hugo-gruson/) and discuss how Continuous Integration can simplify your development process and how to get set up.
+
+And remember, you can always cowork independently on work related to R, work on packages that tend to be neglected, or work on what ever you need to get done!
+
```
Comment (yabellini) [inline content/blog/2022-11-18-ropensci-news-digest-november-2022/index.Rmd:100]:
```suggestion
### rOpenSci communication channels as an alternative to Twitter:
🐘 Mastodon account (if you like social media): https://fosstodon.org/web/@ropensci
💬  Discussion forum: https://discuss.ropensci.org/
🗞️ Newsletter: https://ropensci.org/news/
✍️ Blog: https://ropensci.org/blog/

You can read more on our blog post: https://ropensci.org/blog/2022/11/16/mastodon-en/
```

**Review by yabellini — COMMENTED** [review]
Adding news about twitter

**Pull request merged**